pkgs/development/python-modules/litellm/default.nix

```
@@ -12,15 +12,14 @@
 , certifi
 , appdirs
 , aiohttp
-, httpx
 }:
 let
   version = "1.22.3";
 in
 buildPythonPackage rec {
```

```suggestion
buildPythonPackage {
```
or inline `version`.

---

thank you!